tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -46,6 +46,7 @@ WORKSPACE="${WORKSPACE:-$(upsearch WORKSPACE)}"
 ROOT_CONTAINER=${ROOT_CONTAINER:-tensorflow/tensorflow}
 TF_ROOT_CONTAINER_TAG=${ROOT_CONTAINER_TAG:-devel}
 TF_BUILD_VERSION=${TF_DOCKER_BUILD_DEVEL_BRANCH:-master}
+TF_BUILD_VERSION_IS_PR=${TF_DOCKER_BUILD_DEVEL_BRANCH_IS_PR:-no}
```

Comment:
Please add a comment that `TF_BUILD_VERSION` has two meanings: 
- a commit hashtag when `TF_BUILD_VERSION_IS_PR` is `no`
- a PR ID, otherwise.

It might be good to change the name to reflect the fact that it can also be a PR ID. (Not changing is fine too.)